change-notes/1.20/analysis-cpp.md

```diff
@@ -8,6 +8,7 @@
 
 | **Query**                   | **Tags**  | **Purpose**                                                        |
 |-----------------------------|-----------|--------------------------------------------------------------------|
+| Use of string copy function in a condition (`cpp/string-copy-return-value-as-boolean`) | correctness | This query identifies calls to string copy functions used in conditions, where it's likely that a different function was intended to be called. |
```

Comment:
It looks like UsingStrcpyAsBoolean.ql is an existing query, rather than a new one.

Reply:
Why does it look like that? It's new for 1.20.